requirements/static/darwin.in

```diff
@@ -25,7 +25,7 @@ rfc3987
 salttesting==2017.6.1
 strict_rfc3339
 supervisor==3.3.5; python_version < '3'
-virtualenv
+virtualenv==20.0.10
```

Comment:
@s0undt3ch any chance i can ask you again to run pre-commit here again? i really need to figure out why mine is stack tracing.

Reply:
Sure

Reply:
Done.